airflow/contrib/hooks/spark_submit_hook.py

```diff
@@ -375,6 +378,9 @@ def _process_spark_submit_log(self, itr):
                     self.log.info("identified spark driver id: {}"
                                   .format(self._driver_id))
 
+            if self._connection['deploy_mode'] == 'client':
```

Comment:
We have a lot of branching here. Maybe just add the `self.log.info(line)` to the else branch.

Reply:
Good idea.